docs/src/pages/[platform]/getting-started/usage/vite/react.mdx

```diff
@@ -1,9 +1,7 @@
 import { Tabs } from '@aws-amplify/ui-react';
 import { Example, ExampleCode } from '@/components/Example';
 import { BasicShoppingCard, AdvancedShoppingCard } from '../shared';
-import { InstallScripts, TerminalCommand } from '@/components/InstallScripts.tsx';
-import VitePolyfill from '../../troubleshooting/shared/vite-polyfill.mdx';
-import VitePolyfillTS from '../../troubleshooting/shared/vite-polyfill-ts.mdx';
```

Comment:
Note, these polyfills are still used on the troubleshooting page, so I didn't outright delete them.